yaki_admin_backend/src/main/java/com/xpeho/yaki_admin_backend/data/models/UserModel.java

```diff
@@ -165,4 +164,9 @@ public List<CustomerModel> getCustomers() {
     public void setCustomers(List<CustomerModel> customers) {
         this.customers = customers;
     }
+
+    public int getId() {
```

Comment:
```suggestion
    public int getUserId() {
```

Reply:
I delete this getter because it already exists